packages/quantic/force-app/main/default/lwc/quanticDidYouMean/quanticDidYouMean.js

```diff
@@ -100,10 +131,39 @@ export default class QuanticDidYouMean extends LightningElement {
     );
   }
 
+  get showingResultsForLabel() {
+    return I18nUtils.format(
+      this.labels.showingResultsFor,
+      I18nUtils.getTextBold(I18nUtils.escapeHTML(this.newTriggeredQuery))
+    );
+  }
+
```

Comment:
should we leave a comment here to say that this is for e2e tests purposes?

Reply:
This is not for e2e tests purposes, what is making you think that it is?
This just for showing the right label with the query in a bold format.